packages/concerto-cto/lib/parser.pegjs

```diff
@@ -1687,7 +1692,7 @@ FromUri
   = FromToken __ u:$URI __ {
     return u;
   }
-
+     
```

Comment:
nitpick: the same goes for this extra whitespace (the new line is fine. )In VSCode you can set to trim trailing whitespaces.

Reply:
Yes, this is a nice shortcut. Thanks